### PR TITLE
E-mail validation

### PR DESCRIPTION
**Pull request by hunter4466**
Added javascript code to validate E-mail through a function.
The function catches the request and holds it until its wether approved or not and returns a boolean.
Added animation styles for pop-up and input blinking.

### CODE REVIEW

File: Assets/Javascript/index.js [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+  }
+  return true;
+}
+if (emailRegExpr < 0) {
+  formValidate();
+}
+
+emailInput.addEventListener('input', (event) => {
+  event.preventDefault();
+  announce.remove();
```
Comment (arslanbisharat) [inline Assets/Javascript/index.js:42]: - [ ] The contact form should contain at least one validation checking if the email is written in all lowercase:

- [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up.


- [ ] The error message should show up close to the "submit" button.
- [ ] The form "submit" or the submit button/input "click" event is used to attach the function checking the email format.
 - [ ] There should be a check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: email === email.toLowerCase(), utilizing regular expressions, or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested,

Hi @hunter4466, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: Assets/Javascript/index.js (hunk 1 above)
Comment (hunter4466) [reply Assets/Javascript/index.js:42]: Hello! Thank you , I really apreciate the time you spent in this code review, but, can you be more specific please?

In response to the first point.- I've actually made an algorihtm that tests the upper and lower case sensitivity to hold form action with a regular expression comparisson, isn't it valid?.

In response to the second point.- A message pop-up with error is shown when user uses uppercase character at the e-mail input.

In response to the third point.- ok, my message is shown near to the e-mail input, so I'll correct that.

In response to the fourth point.- the submit button is actually triggering the validation function set at the javascript index.js file.

In response to the fifth point.- ok I'll do the code for showing the check. Thanks.

Could you please explain to me the first, second, and fourth point issues?

Thank you so much in advanve!

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @hunter4466  , 
 Good job implementing the previously required changes. I like the little error and right signs you kept for the email validation! 👍🏾  Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  Congratulations! 🎉 


**Optional**
- [ ] You have a horizontal overflow on Desktop screens. Kindly fix it.
- [ ] Also, your index.html file has some JavaScript functions in it. Kindly put them in a file and add them to the HTML file using `script`. 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**